examples/MultiHypo3Door.jl

```diff
@@ -305,10 +302,10 @@ ensureAllInitialized!(fg)
 
 
 tree = wipeBuildNewTree!(fg, drawpdf=true, show=true)
-cliqorder = getCliqOrderUpSolve(tree)
```

Comment:
@dehann, do you perhaps know the replacement for deprecated `getCliqOrderUpSolve`

Reply:
`getEliminationOrder`